server.go

```diff
@@ -320,39 +321,42 @@ func updateCheck() error {
 
 	resp, err := http.Get("https://storage.googleapis.com/minikube-gcp-auth/releases.json")
 	if err != nil {
-		return fmt.Errorf("failed to get releases file: %v", err)
+		log.Printf("failed to get releases file: %v", err)
 	}
 	defer resp.Body.Close()
 	if err := json.NewDecoder(resp.Body).Decode(&releases); err != nil {
-		return fmt.Errorf("failed to decode releases file: %v", err)
+		log.Printf("failed to decode releases file: %v", err)
 	}
 	if len(releases) == 0 {
-		return fmt.Errorf("no releases found in releases file")
+		log.Printf("no releases found in releases file")
```

Comment:
We can use `log.Print`, https://pkg.go.dev/log#Print

Reply:
Done